d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
+					font-family: inherit;
+					font-size: 0.8rem;
+					font-weight: 400;
+					line-height: 1rem;
```

Comment:
I try to hate all `line-height` values equally. @margaree how do you feel about `line-height: normal` instead?